app/src/main/java/com/spoony/spoony/core/designsystem/component/topappbar/TagTopAppBar.kt

```diff
@@ -0,0 +1,53 @@
+package com.spoony.spoony.core.designsystem.component.topappbar
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.spoony.spoony.core.designsystem.component.tag.LogoTag
+import com.spoony.spoony.core.designsystem.theme.SpoonyAndroidTheme
+import com.spoony.spoony.core.designsystem.type.TagSize
+
+@Composable
+fun TagTopAppBar(
+    count: Int,
+    content: @Composable () -> Unit = {},
+    tagSize: TagSize = TagSize.Small,
+    showBackButton: Boolean = false,
+    onBackButtonClick: () -> Unit = {}
+) {
```

Comment:
p2) content를 맨 밑으로 둔다면 lambda로 사용할 수 있습니다.
그리고 content에 값이 옵셔널이어야 하나요?

```suggestion
@Composable
fun TagTopAppBar(
    count: Int,
    tagSize: TagSize = TagSize.Small,
    showBackButton: Boolean = false,
    onBackButtonClick: () -> Unit = {},
    content: @Composable () -> Unit = {}
) {
```